kotlin-react-router-dom/src/main/kotlin/react/router/dom/routingDsl.kt

```diff
@@ -13,7 +13,7 @@ fun RBuilder.hashRouter(
     basename: String = "",
     hashType: HashType = HashType.slash,
     getUserConfirmation: GetUserConfirmation? = null,
-    handler: RHandler<RProps>,
+    handler: RHandler<PropsWithChildren>,
```

Comment:
`Props`? +4 else handlers

Reply:
Or even `RHandler<*>`?

Reply:
I think it is correct change, because later it will be just `PropsWithChildren.() -> Unit`.
Of course, `hashRouter` and other routers have children, [see docs](https://reactrouter.com/web/api/Router):
```js
ReactDOM.render(
  <Router history={history}>
    <App />
  </Router>,
  node
);
```

Reply:
Oh, think I've got you idea, we don't need `attrs` builder at all here, I will perform corrections

Reply:
`attrs` for `key` and `ref` can be used

Reply:
fixed

Reply:
Yea, you are write about `key` and `ref`, but `RHandler<Props>` is not suitable here because we should be able to setup children in builder

Reply:
Shall I just revert my last commit?

Reply:
> RHandler<Props> is not suitable here because we should be able to setup children in builder

Children will work with `Props` :)